### PR TITLE
Fixed "Discovery screen is displayed after closing option screen"

### PR DESCRIPTION
**Pull request by sveeravelli**

### CODE REVIEW

**Comment by sveeravelli** [comment]
Start screen error for next video is also added to same PR.

File: sdk/react/ooyalaSkinCore.js [hunk 1]
```diff
@@ -191,7 +191,7 @@ OoyalaSkinCore.prototype.onCurrentItemChange = function(e) {
     width:e.width,
     height:e.height,
     captionJSON:null});
-  if (!this.skin.state.autoPlay) {
+  if (this.skin.state.autoPlay != false) {
```
Comment (lental) [inline sdk/react/ooyalaSkinCore.js:194]: really? this is  the bug for the start screen?

Comment (Matthew-C) [inline sdk/react/ooyalaSkinCore.js:194]: Isn't this reverse the logic?

**Comment by lental** [comment]
created https://github.com/ooyala/ios-skin/pull/377, which does not have the start screen fix.  I'm going to accept that, and close this

**Pull request closed without merging**